specs/UserActivityHistory/UserActivityHistoryAPI.md

```diff
@@ -0,0 +1,243 @@
+UserActivityHistory
+===
+
+# Background
+
+The UserActivity class can be used to note down and preserve a record of activities that the user
+is currently doing on their computer - e.g., browsing a website, reading a Word document, etc.
+
+To record user activity, you use UserActivityChannel to retrieve a UserActivity object via the
+API GetOrCreateUserActivityAsync. If a UserActivity with the given ID already exists, it will be
+returned; otherwise, a new UserActivity object will be created and returned. You can then call
+the API GetSession to return a UserActivitySession object that tracks how long the user is engaged
+in that activity. This structure allows multiple sessions to be associated with the same activity,
+representing the case where the user completes that activity a bit at a time - e.g., beginning to
+watch a movie, then pausing, then watching more later. These will be treated as the same singular
+user activity that spans multiple sessions.
+
+UserActivityHistory is a new set of APIs that allow you to query the past 28 days of the user's
+activity history, which will enable you to bring back content that the user has previously been
+interacting with.
+
+# API Pages
+
+## UserActivityHistory class
+
+This class provides static methods that enable you to query the user's activity history.
+This activity history is stored in a database managed by a local service, and these APIs
+call out to that service to retrieve data from the database.
+
+Here is an example usage of the class that will enable you to bring back the webpage for a
+Korean recipe that the user had previously interacted with within the last day:
+
+```c#
+UserActivityHistoryQuery query = new();
+query.Keywords = new string[] { "Korean", "recipe" };
```

Comment:
can I just pass "Korean recipe" instead? what is the criteria for choosing to wordbreak or not?

Reply:
"Korean recipe" would match those exact words in that order with the space in the middle.  Passing in the two separately will match any display text containing "Korean" and "recipe".  So "A recipe for Korean food", "A Korean bibimbap recipe" would match "Korean" and "recipe", but would not match "Korean recipe".